user/views.py

```diff
@@ -1,3 +1,102 @@
-from django.shortcuts import render
+import json, bcrypt
+import re
+
+from django.views import View
+from django.http import JsonResponse
+from .models import User, Gender, Grade, TermsAndCondition, Address
+from my_settings import SECRET_KEY
 
 # Create your views here.
+class SignupView(View): 
+    def validate_account(self, account): # 6자 이상의 영문 또는 영문과 숫자를 조합 
+        REGEX_ACCOUNT_1 = '^[a-z]{6,}$'                      # 영문 6자 이상
+        REGEX_ACCOUNT_2 = '^(?=.*[0-9])(?=.*[a-z]).{6,}$'    # 영문+숫자 6자 이상
+
+        if not re.match(REGEX_ACCOUNT_1, account) and not re.match(REGEX_ACCOUNT_2, account):
+            return False
+        return True
```

Comment:
```suggestion
        return re.match(REGEX_ACCOUNT_1, account) and not re.match(REGEX_ACCOUNT_2, account):
```

파이썬에서는 반환 되는 값 자체가 boolean 일 경우나, Truthy, Falsy 한 값일 경우,  == False == None == 0등으로 평가하지 않고, 그 값을 그대로 사용합니다. 그 자체로도 True or False를 판단할 수 있기 때문입니다. False한 지 평가하실 때는 not 을 활용해보세요 :)

Reply:
아래 다른 코드들도 마찬가지입니다.